tests/test_cli.py

```diff
@@ -150,18 +153,21 @@ def test_usage_string(tmpdir, capsys):
 
     assert EXPECTED_USAGE in out
 
+@pytest.mark.xfail(platform.system() == "Windows" and six.PY3,
+                   reason="Windows encoding problems")
 def test_usage_string_fork(tmpdir, capsys):
     tmpdir.chdir()
 
     try:
-        out = check_output('bumpversion --help', shell=True, stderr=subprocess.STDOUT).decode('utf-8')
+        out = check_output('bumpversion --help', shell=True, stderr=subprocess.STDOUT)
```

Comment:
There is a `universal_lines` parameter which also fixes encoding issues. Perhaps that'd remove the need for `xfail`?

Reply:
I believe it's `universal_newlines` FWIW